boa/src/builtins/function/mod.rs

```diff
@@ -18,84 +18,43 @@ use crate::{
         value::{RcString, Value},
         Array,
     },
-    environment::function_environment_record::BindingStatus,
-    environment::lexical_environment::{new_function_environment, Environment},
-    exec::{Executable, Interpreter},
+    environment::lexical_environment::Environment,
+    exec::Interpreter,
     syntax::ast::node::{FormalParameter, StatementList},
     BoaProfiler, Result,
 };
 use bitflags::bitflags;
 use gc::{unsafe_empty_trace, Finalize, Trace};
 use std::fmt::{self, Debug};
 
-/// _fn(this, arguments, ctx) -> Result<Value>_ - The signature of a built-in function
-pub type NativeFunctionData = fn(&Value, &[Value], &mut Interpreter) -> Result<Value>;
+/// _fn(this, arguments, ctx) -> ResultValue_ - The signature of a built-in function
```

Comment:
I think this comment needs to be updated back again.
```suggestion
/// _fn(this, arguments, ctx) -> Result<Value>_ - The signature of a built-in function
```